src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -286,7 +286,6 @@ internal override void OnLayoutUpdated()
 				if (_rootEltUsedToProcessScrollTo is null && rootElement is FrameworkElement rootFwElt)
 				{
 					_rootEltUsedToProcessScrollTo = rootFwElt;
-					rootFwElt.LayoutUpdated += TryProcessScrollTo;
 				}
```

Comment:
This has been added for the `CalendarView` that was requesting to scroll to a given offset even before the `ScrollViewer` has been added into the visual tree. (It has been has been added by this commit https://github.com/unoplatform/uno/commit/819e28e6a628e4fd2ae49fd41f9f2c25e2bc2371 which is part of the massive `CalendarView` PR : https://github.com/unoplatform/uno/pull/6098).

It might not be needed with your last changes, but I'm not sure that there are tests for this ... you should check manually if there is nothing broken (on at least a chromium based browser and Firefox).

Reply:
Let's keep for now then and revisit if proven to be problematic